tests/presubmit/presubmit.sh

```diff
@@ -2,7 +2,14 @@
 set -ex
 # shellcheck source=/dev/null
 source "$KOKORO_GFILE_DIR/common.sh"
+sudo chmod +x "$KOKORO_GFILE_DIR/verify-commits.sh"
+"$KOKORO_GFILE_DIR"/verify-commits.sh
+
+# Grab the latest version of shellcheck and add it to PATH
+wget https://storage.googleapis.com/shellcheck/shellcheck-latest.linux.x86_64.tar.xz
```

Comment:
Should we use allow_file instead of this? Then we could have it already unpacked.

Reply:
yeah that makes sense since it's in GCS